x/bridge/observer/osmosis/test_events/tx_check_success.json

```diff
@@ -0,0 +1,13 @@
+{
+    "code": 0,
+    "data": null,
+    "log": "",
+    "info": "",
+    "gas_wanted": "200000",
+    "gas_used": "39377",
+    "events": [],
+    "codespace": "sdk",
+    "sender": "",
+    "priority": "0",
```

Comment:
Where did you get this example tx check success from? I don't think we have the priority field but I could be wrong.